x-pack/legacy/plugins/canvas/public/state/reducers/elements.js

```diff
@@ -29,14 +31,14 @@ function getNodeIndexById(page, nodeId, location) {
 function assignNodeProperties(workpadState, pageId, nodeId, props) {
   const pageIndex = getPageIndexById(workpadState, pageId);
   const location = getLocationFromIds(workpadState, pageId, nodeId);
-  const nodesPath = ['pages', pageIndex, location];
+  let nodesPath = `pages.${pageIndex}.${location}`;
   const nodeIndex = get(workpadState, nodesPath, []).findIndex(node => node.id === nodeId);
 
   if (pageIndex === -1 || nodeIndex === -1) {
     return workpadState;
   }
 
-  return assign(workpadState, nodesPath.concat(nodeIndex), props);
+  return assign(workpadState, (nodesPath += '.' + nodeIndex), props);
```

Comment:
NIT: The inline assignment of `nodesPath` seems unnecessary.  Replace with a template literal maybe? 

` ${nodesPath}.${nodesIndex}` reads much better?